webserver/views/datasets.py

```diff
@@ -227,6 +227,8 @@ def evaluate(dataset_id):
             flash.error("Cannot add this dataset because of a validation error: %s" % e)
         except db.dataset_eval.JobExistsException:
             flash.warn("An evaluation job for this dataset has been already created.")
+        if len(ignored_rec) > 0:
+            flash.warn("The following recording are ignored: %s" % ', '.join(ignored_rec))
```

Comment:
I think it'd be a good idea to include more information in this message. At the very least I'd like to see the classes for each recording.
A great additional feature would be to use our metadata endpoint to also get the recording title and artist: 
https://github.com/metabrainz/acousticbrainz-server/blob/607167ba1078df7853dc7b498fa5c706b4cb3c72/webserver/views/datasets.py#L393

We have some react code for the dataset editor to do this, I'm not sure if we can reuse it on this page or if it's better to duplicate the code:
https://github.com/metabrainz/acousticbrainz-server/blob/f9afa4017c49b7d412b6557b68744d8fbbb2da07/webserver/static/scripts/datasets/editor.js#L598

Reply:
I added the classes information to the output.  Regarding the metadata endpoint, I'm planning to work on that in separate PR.